doc/rst/language/spec/tuples.rst

```diff
@@ -756,43 +725,58 @@ where a comma-separated list of components is valid.
 Value Tuples and Referential Tuples
 -----------------------------------
 
-Throughout the next few sections, the terms referential tuple and value
-tuple are used frequently to describe two different ways that tuples can
-capture elements.
+The terms referential tuple and value tuple describe two different ways
+that tuples capture their elements.
 
-Tuple expressions or tuple arguments with default argument intent are two
-examples of referential tuples. They store elements by reference where it
-makes sense to do so. Referential tuples may be viewed as analogous to a
-group of function arguments that each have default argument intent.
+A *value tuple* stores all its elements by value and may store elements
```

Comment:
I would choose either "element" or "component" here (I think I prefer the term component).

Reply:
We call them "components", so I switched everything to that.